sycl/plugins/rocm/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+message(STATUS "Including the PI API ROCM backend.")
+
+ # cannot rely on cmake support for ROCM; it assumes runtime API is being used.
+ # we only require the ROCM driver API to be used
+ # rocm_rocm_LIBRARY variable defines the path to libhsa-runtime64.so, the ROCM Driver API library.
+
+#find_package(ROCM 4.0 REQUIRED)
+
+# Make imported library global to use it within the project.
+add_library(rocmdrv SHARED IMPORTED GLOBAL)
+
+
+set(ROCM_ROCM_LIBRARY "/opt/rocm/hip/lib/libamdhip64.so")
+set(ROCM_INCLUDE_DIRS "/opt/rocm/hip/include")
```

Comment:
NIT. Minor. Probably these paths should be configurable from outside CMake file.